native-image-logging-examples/README.md

```diff
@@ -0,0 +1,92 @@
+# Native Image Logging Configuration Examples
+
+This repository contains the code for a demo application for [GraalVM](http://graalvm.org).
+
+### Prerequisites
+* [GraalVM](http://graalvm.org)
+* [Native Image](https://www.graalvm.org/docs/reference-manual/native-image/)
+
+## Preparation
+
+1. [Download GraalVM](https://www.graalvm.org/downloads/), unzip the archive, export the GraalVM home directory as the `$GRAALVM_HOME` and add `$GRAALVM_HOME/bin` to the `PATH` environment variable.
+
+  On Linux:
+  ```bash
+  export GRAALVM_HOME=/home/${current_user}/path/to/graalvm
+  export PATH=$GRAALVM_HOME/bin:$PATH
+  ```
+  On macOS:
+  ```bash
+  export GRAALVM_HOME=/Users/${current_user}/path/to/graalvm/Contents/Home
+  export PATH=$GRAALVM_HOME/bin:$PATH
+  ```
+  On Windows:
+  ```bash
+  setx /M GRAALVM_HOME "C:\Progra~1\Java\<graalvm>"
+  setx /M PATH "C:\Progra~1\Java\<graalvm>\bin;%PATH%"
+  ```
+
+2. Install [Native Image](https://www.graalvm.org/docs/reference-manual/native-image/#install-native-image):
+
+  ```bash
+  gu install native-image
+  ```
+
+3. Download or clone the repository and navigate into the `native-image-logging-examples` directory:
+  ```bash
+  git clone https://github.com/graalvm/graalvm-demos
+  cd graalvm-demos/native-image-logging-examples
+  ```
+
+There are two Java classes, one for the build-time logger initialization and second for runtime logger initialization. The logger will be initialized with a custom _logging.properties_ configuration file, which is placed in the same directory as _BuildTimeLoggerInit.java_ and _RuntimeLoggerInit.java_.
```

Comment:
Update the sentence to: 

There are two Java classes: one for the build-time logger initialization and the second for runtime logger initialization.